# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436886266 14400
#      Tue Jul 14 11:04:26 2015 -0400
# Node ID ff56b738854c5a8c06afde0bca9f874c63484f3e
# Parent  916cc788d90604253c34658285601c0ec0c14353
don't delete $(BUILT_DISTFILES) for maintainer-clean target

* Makefile.am (MAINTAINERCLEANFILES): Don't add $(BUILT_DISTFILES) to
the list.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -153,19 +153,16 @@ OCTAVE_INTERPRETER_TARGETS += run-octave
 CLEANFILES += \
   $(BUILT_SOURCES)
 
 DISTCLEANFILES += \
   .gdbinit \
   ChangeLog \
   $(DIRSTAMP_FILES)
 
-MAINTAINERCLEANFILES += \
-  $(BUILT_DISTFILES)
-
 CONFIG_FILES = @ac_config_headers@ @ac_config_files@
 
 octinclude_HEADERS += oct-conf-post.h
 nodist_octinclude_HEADERS += config.h
 
 all-local: $(ALL_LOCAL_TARGETS) $(noinst_SCRIPTS) $(DIST_INFO_FILES) .gdbinit $(DOC_TARGETS)
 	@echo ""
 	@echo "Octave successfully built.  Now choose from the following:"
